core/src/main/resources/org/apache/spark/ui/static/jquery.cookies.2.2.0.min.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) 2005 - 2010, James Auldridge
```

Comment:
are the cookies used anywhere? Can you test to see if we can remove it, if we can't where is it called from?